README.md

```diff
@@ -2,18 +2,26 @@ Please see our <a href="https://github.com/tenjin/tenjin-unity-sdk/wiki">Release
 
 For any issues or support, please contact: support@tenjin.com
 
-Tenjin Unity plugin
+Tenjin Unity
 =========
-- Allows unity developers to quickly integrate with Tenjin's install API
-- Review the [iOS](https://github.com/tenjin/tenjin-ios-sdk) and [Android](https://github.com/tenjin/tenjin-android-sdk) documentation and apply the proper platform settings to your builds. Most importantly:
-  1. **iOS**: make sure you have the right build settings and you include the iOS frameworks you need.
-  2. **Android**: If you have another SDK installed which already has Google Play Services installed or uses [PlayServicesResolver](https://github.com/googlesamples/unity-jar-resolver), you may need to delete this file: `/Assets/Plugins/Android/play-services-basement-11.0.4.aar`
-  3. Your "API_KEY" is located on your [Organizations tab](https://www.tenjin.io/dashboard/organizations)
+* Tenjin Unity SDK supports both iOS and Android.
+* Your "API_KEY" is located on your [Organizations tab](https://www.tenjin.io/dashboard/organizations)
+* Review the [iOS](https://github.com/tenjin/tenjin-ios-sdk) and [Android](https://github.com/tenjin/tenjin-android-sdk) documentation and apply the proper platform settings to your builds.
+* **iOS Notes**:
+  1. Xcode 12 is required if using Unity iOS SDK v1.12.0 and higher.
```

Comment:
Does this statement apply to Unity as well? Just wanted to double check, but I'm guessing Xcode needs to be installed in order to run/build Unity iOS apps locally?

Reply:
yeah, Unity compiles to a Xcode project